smart-contracts/osmosis/contracts/cl-vault/tests/test-tube/setup.rs

```diff
@@ -585,7 +587,7 @@ pub fn calculate_deposit_ratio(
     };
 
     // TODO: Compute this based on tokens_provided size
-    let ratio_approx: String = "0.00005".to_string();
+    let ratio_approx: String = "0.02".to_string();
```

Comment:
I recall we spoke about this once, but nothing more. What's the reason for this big change?

Reply:
I've been able to revert this to the original value after splitting 50% 50% and tweaking a bit the deposit values. I also deprecated it from the setup.rs and inlined it in the autocompounds.rs which is the only place that needs it.